File: RxUI/src/main/java/com/vondear/rxui/view/RxRunTextView.java
package com.vondear.rxui.view;

import android.content.Context;
import android.util.AttributeSet;
import android.view.ViewDebug;

import androidx.appcompat.widget.AppCompatTextView;

/**
 * @author vondear
 * @date 2016/6/28
 */
public class RxRunTextView extends AppCompatTextView {
    public RxRunTextView(Context context, AttributeSet attrs, int defStyle) {
        super(context, attrs, defStyle);
    }

    public RxRunTextView(Context context, AttributeSet attrs) {
        super(context, attrs);
    }

    public RxRunTextView(Context context) {
        super(context);
    }

    /**
     * 当前并没有焦点，我只是欺骗了Android系统
     */
    @Override
    @ViewDebug.ExportedProperty(category = "focus")
    public boolean isFocused() {
        return true;
    }
}

File: RxUI/src/main/java/com/tamsiree/rxui/view/RxRunTextView.kt
package com.tamsiree.rxui.view

import android.content.Context
import android.util.AttributeSet
import android.view.ViewDebug.ExportedProperty
import androidx.appcompat.widget.AppCompatTextView

/**
 * @author tamsiree
 * @date 2016/6/28
 */
class RxRunTextView : AppCompatTextView {
    constructor(context: Context, attrs: AttributeSet?, defStyle: Int) : super(context, attrs, defStyle)
    constructor(context: Context, attrs: AttributeSet?) : super(context, attrs)
    constructor(context: Context) : super(context)

    /**
     * 当前并没有焦点，我只是欺骗了Android系统
     */
    @ExportedProperty(category = "focus")
    override fun isFocused(): Boolean {
        return true
    }
}